l3build-check.lua

```diff
@@ -556,7 +556,7 @@ local function normalize_pdf(content)
       not match(line,"^%%%%%+") then
       line = gsub(line,"%/ID( ?)%[<[^>]+><[^>]+>]","/ID%1[<ID-STRING><ID-STRING>]")
       line = gsub(line,"%/ID( ?)%[(%b())%2%]","/ID%1[<ID-STRING><ID-STRING>]")
-      line = gsub(line,"GhostScript %d%.%d+%.?%d*","GhostScript ...")
+      line = gsub(line,"Ghost([sS])cript %d+%.%d+%.?%d*","Ghost%1cript ...")
```

Comment:
If we are generalising, shouldn't the output version here be fixed?

Reply:
You mean always substituted with one of uppercase or lowercase s, but not both?

At first I do want to use only "Ghostscript ...", because it's used by recent gs, checked with gs 10.00.0 on my macOS, 9.55.0 on github actions ubuntu LTS 22.04, 9.56.1 in @u-fischer's `pdfresources` repo whose testfiles are generated on windows from my guess.  I also guess when you implemented it about one year ago, at that time gs did write uppercased S.

It's the fear of breaking existing testfiles that makes me step back.  One can also argue the case of S is not part of the gs version.

Reply:
Who knows what the pattern in future versions of entry `/Producer(GPL Ghostscript 9.55.0)` (the one generated by gs on ubuntu LTS 22.04) will be changed to... Will `/Producer\(.*Ghost[sS]cript.*\)` be too general? A line by line matching is too limited.

Ah wait, Is the `/Producer` entry overwritten-able?

Reply:
I meant that we already have `GhostScript` and `Ghostscript`, so normalising those two as well as the version number would be good.

Reply:
haha I was complicating things...